layers/gpuav/spirv/module.cpp

```diff
@@ -489,7 +490,9 @@ void Module::LinkFunction(const LinkInfo& info) {
             // for simplicity, just create a new constant for things other than 32-bit OpConstant as there are rarely-to-none
             // composite/null/true/false constants in linked functions. The extra logic to try and find them is much larger and cost
             // time failing most the searches.
-            if (opcode == spv::OpConstant) {
+            //
+            // If length is 5, it is a 64-bit constant, which we don't care about
```

Comment:
```suggestion
            // If length is 4, it is a 64-bit constant, which we don't care about
```
or should the check be `new_inst->Length() == 4`?

Reply:
- length() == 4 == 32-bit
- length() == 5 == 64-bit

I only want to check for 32 bits, will just adjust it to be `new_inst->Length() < 5`